fs/fsutils/fsutils_unix.go

```diff
@@ -0,0 +1,37 @@
+// +build !windows
+
+package fsutils
```

Comment:
I think this could probably just belong in `mountinfo`. We intend to move the `fs` package to continuity and don't want to import into other `containerd/containerd` packages.